src/core/column/isna.h

```diff
@@ -55,14 +52,13 @@ class Isna_ColumnImpl : public Virtual_ColumnImpl {
 
 
     bool get_element(size_t i, int8_t* out) const override {
-      T tmp;
-      *out = ! arg_.get_element(i, &tmp);
+      *out = !arg_.get_element_validity(i);
```

Comment:
It is not really efficient to use `get_element_validity()` here, see https://github.com/h2oai/datatable/blob/main/src/core/column.h#L173-L181